### ISSUE TITLE
Huge issues downloading Flutter five days and still not complete.

### ISSUE THREAD

**Issue by flutternooob** [flutter/flutter#72180]
Hi. I'm just new on here and new to coding. I was hoping to start using Flutter, but ive been met with some huge issues. 
I downloaded it and followed the instructions. I updated my path etc etc. I could use flutter in cmd prompt on windows screen. When i downloaded Andriod studio I downloaded the plugins Flutter and Dart. Then Flutter doctor said they were not installed even though I installed them. When I was trying to create a nee Flutter project in the studio it says add the sdk path so I done it and it says something along the lines of a file called Flutter already exists and wouldn't let me go any further. Eventually randomly it did work, then I couldn't find my android phone as a device despite following instructions again. So i deleted the project reset it and now the same issue of a file called flutter already exists is popping up. After trying for so long I deleted everything and started again. Now i cant even use flutter commands. I'm seriously considering deleting it all permanently. I was trying to contact flutter, but they dont seem to have a support number. If anyone can help me it would be much appreciated thanks.

**Comment by ameysunu** [comment]
Hi, did you try running the ``flutter doctor`` command? If that doesn't work try deleting all the flutter files and install from scratch. Follow the installation steps from here: https://flutter.dev/docs/get-started/install . If it still doesn't work, hmu.

**Comment by flutternooob** [comment]
Oh believe me ive ran flutter doctor hundreds of times. Ive deleted and redownloaded all software 5 times now. Its driving me crazy. I followed the instructions on that link too. I can't figure it out what the problem is.

**Comment by ameysunu** [comment]
Can you get in touch with me via email, you'll find it on my Github profile? Thanks.

**Comment by flutternooob** [comment]
Is there a way to message you here? Just i used my personal email for this account, and I don't want to use my personal email. I have no other email accounts to use. If thats ok with you. Is there no way I can message you on here? I hope you don't mind.

**Comment by ameysunu** [comment]
Sure! Can you attach a few screenshots displaying your error?

**Comment by flutternooob** [comment]
Ok I'm going to try and uninstall everything and reinstall then I'll get they screenshots and put them up shortly. I'll be back in touch with them when its done. Thanks.

**Comment by iapicca** [comment]
@flutternooob 

> Ok I'm going to try and uninstall everything and reinstall then I'll get they screenshots and put them up shortly. I'll be back in touch with them when its done. Thanks.

please copy paste the logs instead of a screenshot

**Comment by sonuishaq67** [comment]
Hey I am having issues while installing flutter too (on linux mint)
It just gets stuck forever at 
```Building flutter tool...```

**Comment by flutternooob** [comment]
Ok then I'll do copy and paste. > @flutternooob 
> 
> 
> 
> > Ok I'm going to try and uninstall everything and reinstall then I'll get they screenshots and put them up shortly. I'll be back in touch with them when its done. Thanks.
> 
> 
> 
> please copy paste the logs instead of a screenshot
> 
>

**Comment by flutternooob** [comment]
Did you get any further forward? I find googling is unhelpful I've not found one place that shows how to fix issues. Everyone assumes its a smooth download not the case. Mines is totally broke now I can't even type commands its flashing up red saying unblock file. > Hey I am having issues while installing flutter too (on linux mint)
> 
> It just gets stuck forever at 
> 
> ```Building flutter tool...```
> 
> 
> 
> 
> 
> i tried googling this but the previous solutions didnt help 
> 
> 
> 
> #12341 #43066

**Comment by sonuishaq67** [comment]
> Did you get any further forward? 
nope still cant update 
says the same thing i think pub.dev is down for some packages

**Comment by flutternooob** [comment]
> > Did you get any further forward? 
> 
> 
> 
> nope still cant update 
> 
> says the same thing i think pub.dev is down for some packages 

Its been like this for five days for me. It wouldn't be down for five days would it?

**Comment by flutternooob** [comment]
> 
> 
> Sure! Can you attach a few screenshots displaying your error?

Hi I managed to look and I ave an even bigger issue now. I cant even run commands when I type in cmd and run flutter doctor it flasges up red writing saying ''the term unblock file is not reconisged'' and some other stuff but it flashes up and disapears that quick I cant see it all. What do you think ? thanks

**Comment by TahaTesser** [comment]
Closing as duplicate of https://github.com/flutter/flutter/issues/72215
